inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlTableSource.java

```diff
@@ -182,7 +185,8 @@ public MySqlTableSource(
             Properties jdbcProperties,
             Duration heartbeatInterval,
             boolean migrateAll,
-            String inlongMetric) {
+            String inLongMetric,
```

Comment:
Please change it to `inlongXxx`, thanks.

Reply:
done